mark meeting as happening sporadically (#324)
solid/authorization-panel: README.md

Anyone can join the [Authorization Panel chat](https://gitter.im/solid/authorization-panel).

> Online panel meetings occur sporadically on Wednesday, 14:00 UTC at https://meet.jit.si/solid-authorization.

Author: ```suggestion
Online panel meetings occur sporadically on Wednesday, 14:00 UTC at https://meet.jit.si/solid-authorization.
They are announced and can be requested on the Authorization Panel chat.
```

Reviewer: Sounds good :)

Anyone can join the [Authorization Panel chat](https://gitter.im/solid/authorization-panel).

Online panel meetings occur sporadically on Wednesday, 14:00 UTC at https://meet.jit.si/solid-authorization.

> They are announced and can be requested on the Authorization Panel chat.

Author: ```suggestion
```

Anyone can join the [Authorization Panel chat](https://gitter.im/solid/authorization-panel).

> Online panel meetings are currently paused [and their future is being discussed in PR#322](https://github.com/solid/authorization-panel/pull/322)

Reviewer: ```suggestion
Online panel meetings occur sporadically on Wednesday, 14:00 UTC at https://meet.jit.si/solid-authorization.

The [time and focus](https://github.com/solid/authorization-panel/issues/325) of authorization meetings is currently being discussed.
```

Anyone can join the [Authorization Panel chat](https://gitter.im/solid/authorization-panel).

Online panel meetings occur sporadically on Wednesday, 14:00 UTC at https://meet.jit.si/solid-authorization.

> They are announced and can be requested on the Authorization Panel chat.

Reviewer 2: Live/instant-chat-space is a terrible calendaring mechanism, not least because (as has been expressed on recent minutes) keeping track of all the various conversation spaces (Gitter, Twitter, IRC, GitHub, Slack, email, etc.) is difficult if not impossible.

Meetings should be announced via a calendaring tool such as the W3 Calendar.

Requests would probably be better via a more typically asynchronous tool, such as email or github issue, with associated polling mechanism, to help make chosen timing more convenient for larger numbers of potential attendees.

Author: Ok, I retract this suggestion.

Anyone can join the [Authorization Panel chat](https://gitter.im/solid/authorization-panel).

> Online panel meetings occur sporadically on Wednesday, 14:00 UTC at https://meet.jit.si/solid-authorization.

Reviewer 2: ```suggestion
Online panel meetings occur sporadically on Wednesday, 14:00 UTC at https://meet.jit.si/solid-authorization.

For now, a meeting may be requested by raising an issue on the 
[`authorization-panel` repo](https://github.com/solid/authorization-panel/issues/) 
or posting to the [Solid CG mailing list](mailto:public-solid@w3.org).

```

Reviewer: I would rather keep issues to their current and primary use and I don't follow the Solid CG mailing list.

Links to HackMD and discussions about occurence of meetings have always taken place on Gitter until now, I don't see a very compelling reason to change the way things have worked until now. Especially for a temporary measure.

Author: I honestly doubt that anyone will request the meeting, especially since we want to resolve the stable schedule ASAP.

I suggest merging the PR as it is and if @TallTed feels strongly about this amendment it could be proposed in a separate PR.

Reviewer: Sounds good to me @elf-pavlik .
